src/subdomains/generic/user/models/kyc/kyc-webhook.service.ts

```diff
@@ -89,9 +92,17 @@ export class KycWebhookService {
 
         await this.http.post(`${user.wallet.apiUrl}/kyc/update`, data, {
           headers: { 'x-api-key': apiKey },
+          timeout: 5000,
+          tryCount: 3,
         });
       } catch (error) {
-        console.error(`Exception during KYC webhook (${result}) for user ${userData.id}:`, error);
+        await this.notificationService.sendMail({
+          type: MailType.ERROR_MONITORING,
+          input: {
+            subject: 'Webhook failed',
+            errors: [`Exception during KYC webhook (${result}) for user ${userData.id}:`, error],
+          },
+        });
```

Comment:
Not related to this change but it would generally be good to also log (and have in mail message) the user ID (not only user data ID).